pipelines/HSC/DRP-Prod.yaml

```diff
@@ -0,0 +1,96 @@
+
+      tasks aggregate all detectors for a given visit and perform
+      full focal plane background estimation.
+  step2b:
+    subset:
+      - isolatedStarAssociation
+      - jointcal
+    description: >
+      Tract-level tasks. Permissible dataquery constraints: tract
+
+      jointcal and isolatedStarAssociation use PreSources, generated
+      by consolidatePreSourceTable, for all visits that overlap a tract.
+      jointcal produces solutions per-tract, isolatedStarAssociation produces
+  step2c:
+    subset:
+      - fgcmBuildStarsTable
+      - fgcmFitCycle
+      - fgcmOutputProducts
+      - makeCcdVisitTable
+      - makeVisitTable
+    description: >
+      global-level tasks that must not be run with any dataquery constraints
+      Can be run anytime after subset step2a.
+
+      Permissible dataquery constraints: None
+
+      Tasks generate one data product per collection.
+      FGCM performs better the more visits are included and
+      make[Ccd]VisitTable produces per-collection summary of the Visits
+      and CcdVisits.
+  step2d:
+    subset:
+      - finalizeCharacterization
+      - writeRecalibratedSourceTable
+      - transformSourceTable
+      - consolidateSourceTable
+    description: >
+      visit-level tasks. Permissible dataquery constraints: visit
+
+      writeRecalibratedSourceTable, transformSourceTable run per-detector
+      consolidateSourceTable produces one data product per visit.
+      finalizeCharacterization will eventually model full focal plane
+      PSFs.
+  step3:
+    subset:
+      - makeWarp
+      - assembleCoadd
+      - detection
+      - mergeDetections
+      - deblend
+      - measure
+      - mergeMeasurements
+      - forcedPhotCoadd
+      - writeObjectTable
+      - transformObjectTable
+      - consolidateObjectTable
+      - healSparsePropertyMaps
+    description: >
+      tract-level tasks.  Permissible dataquery constraints: tract
+
+      Tasks that can be run together, but only after the 'step1', 'step2a',
+      'step2b', 'step2c', 'step2d' subsets.
+
+      If you do not include tract in the data query, you will get tracts with
+      partial coverage.
+  step4:
+    subset:
+      - forcedPhotCcd
+    description: >
+      detector-level tasks.  Permissible dataquery constraints: visit, detector
+
+      These detector-level tasks should not be run with 'tract' or 'patch' as
+      part of the data ID expression if all reference catalogs for seeding
+      forced photometry cover these detector-level quanta are desired.
+  step7:
+    subset:
+      - consolidateHealSparsePropertyMaps
+    description: >
+      Tasks that should be run as the final step that require global inputs,
+      and can be run after the 'step3' subset.
+
+      This step has global aggregation tasks to run over all visits, detectors,
+      tracts, etc.  This step should be run only with the instrument constraint
+      in the data query.
```

Comment:
```suggestion
      in the dataquery.
```

Reply:
Is providing instrument in the dataquery same as "Permissible dataquery constraints: None"?